src/data/integration.ts

```diff
@@ -52,3 +52,6 @@ export const fetchIntegrationManifest = (
 
 export const fetchIntegrationSetups = (hass: HomeAssistant) =>
   hass.callWS<IntegrationSetup[]>({ type: "integration/setup_info" });
+
+export const scanUSBDevices = (hass: HomeAssistant) =>
```

Comment:
Should this be in integration? I guess it would make more sense as a separate file `data/usb`

Reply:
Will move it later today

Reply:
done